DirectEmployers/MyJobs
# [PD-3492][PD-3593] Add title, created/updated tracking to PartnerAssessments

## What's this PR do?
Adds custom title, created (by, on) updated (by, on) handling to Partner Assessments.

## Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/29709626/29185877-ac90013c-7dd8-11e7-9712-309323108043.png)

## Where should the reviewer start?
* Run migrations.

## How should this be manually tested?
* Navigate to Partner Assessments
* Add a new assessment with a custom title.
* do `dkm shell` from your command line
* execute the following commands
```
import pprint
from mypartners.models import PartnerAssessment
pa = PartnerAssessment.objects.all().last()
pprint.pprint(pa.__dict__)
```
* Make sure those values look correct.
